a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html

```diff
@@ -296,15 +296,14 @@ <h5>Variables setup</h5>
                     </ol>
                     <h5>Instructions</h5>
                     <ol class="condition">
-                                <li>
-                                    <span><b>IF</b> <var>textual_alternative_images</var>:</span>
-                                    <span><b>THEN</b> display <code id="ways-of-reading-nonvisual-reading-alt-text">"Has alternative text"</code>.</span>
-                                </li>
-
                         <li>
                             <span><b>IF</b> <var>all_necessary_content_textual</var>:</span>
                             <span><b>THEN</b> display <code id="ways-of-reading-nonvisual-reading-readable">"Readable in read aloud or dynamic braille"</code>.</span>
                         </li>
+                        <li>
+                            <span><b>IF</b> <var>textual_alternative_images</var>:</span>
+                            <span><b>THEN</b> display <code id="ways-of-reading-nonvisual-reading-alt-text">"Has alternative text"</code>.</span>
+                        </li>
                         <li>
                             <span><b>ELSE IF</b> <var>non_textual_content_images</var> <b>AND</b> <b>NOT</b> <var>textual_alternative_images</var>:</span>
                             <span><b>THEN</b> display <code id="ways-of-reading-nonvisual-reading-not-fully">"Not fully readable in read aloud or dynamic braille"</code>.</span>
```

Comment:
Putting the IF test here turns it into a new set of if/else statements.

The ONIX techniques make this IF a subtest so it's only output if all_necessary_content_textual is true.

If the value should be output no matter the overall readability in braille or using reading aloud, it should be a separate IF after this series of if/else tests. (That would need fixing in both docs.)